scripts/bluepill.sh

```diff
@@ -82,6 +82,16 @@ bluepill_build()
           exit 1
   }
   set +o pipefail
+  # package bluepill
```

Comment:
I think it was because there were no tags in your tree. You need to `git pull --tags`. At any rate, I added the `--always` flag to `git describe` which makes it fall back to the SHA.